people-and-planet-ai/image-classification/requirements.txt

```
@@ -1,3 +1,3 @@
 pillow==10.3.0
 apache-beam[gcp]==2.55.1
-google-cloud-aiplatform==1.47.0
+google-cloud-aiplatform==1.90.0
```

```suggestion
google-cloud-aiplatform==1.47.0
```

revert update for package failing for unrelated reasons